integrated_channels/utils.py

```diff
@@ -25,6 +26,23 @@
 LOGGER = getLogger(__name__)
 
 
+def encode_course_key_for_lms(edx_course_key):
```

Comment:
Suggestion:  You don't need to specify `for_lms` and `from_lms` here -- I would instead say `to_base64` and `from_base64`.  Whether or not these utils are used for encoding keys for passing to/from an LMS is sort of irrelevant.

Reply:
done, also added error detection now.. don't want to inadvertently process spaces only kinds of inputs since they translate to the same base64 value each time. I hope we dn't have such bad content keys but who knows right?

Reply:
Do we need to be so specific regarding the invalidity of the key?  You could probably just do a more general ValueError try/except around the actual base64 conversion and log-out the issue instead of packing it into the return statement.

Reply:
the only issue there is the '  ' case. That one always encodes successfully to a fixed string .. so wanted to avoid that. Although I could trim that before sending to base64 then we can also do what you mentioned.